demos/theme/index.html

```diff
@@ -800,7 +800,7 @@ <h3 class="mdc-typography--headline5 demo-component-section__heading">
           </p>
         </figure>
         <figure class="demo-text-field-wrapper">
-          <div class="mdc-text-field mdc-text-field--box demo-text-field">
+          <div class="mdc-text-field demo-text-field">
```

Comment:
The input and helper text under this text field still has an id that says box, do we want to update that?